ledger-api/grpc-definitions/com/daml/ledger/api/v1/transfer_command.proto

```diff
@@ -0,0 +1,192 @@
+    google.protobuf.Duration deduplication_time = [deprecated = true];
+
+    // Specifies the length of the deduplication period.
+    // It is interpreted relative to the local clock at some point during the submission's processing.
+    // Must be non-negative. Must not exceed the maximum deduplication time (see
+    // ``ledger_configuration_service.proto``).
+    google.protobuf.Duration deduplication_duration = ;
+
+    // Specifies the start of the deduplication period by a completion stream offset (exclusive).
+    // Must be a valid LedgerString (as described in ``ledger_offset.proto``).
+    string deduplication_offset = ;
+  }
+
+  // TODO are we sure we do not need ledger_time bounds for transfers?
+  // Lower bound for the ledger time assigned to the resulting transaction.
+  // Note: The ledger time of a transaction is assigned as part of command interpretation.
+  // Use this property if you expect that command interpretation will take a considerate amount of time, such that by
+  // the time the resulting transaction is sequenced, its assigned ledger time is not valid anymore.
+  // Must not be set at the same time as min_ledger_time_rel.
+  // Optional
+  // google.protobuf.Timestamp min_ledger_time_abs = 10;
+
+  // Same as min_ledger_time_abs, but specified as a duration, starting from the time the command is received by the server.
+  // Must not be set at the same time as min_ledger_time_abs.
+  // Optional
+  // google.protobuf.Duration min_ledger_time_rel = 11;
+
+  // Set of parties on whose behalf the command should be executed.
+  // If ledger API authorization is enabled, then the authorization metadata must authorize the sender of the request
+  // to act on behalf of each of the given parties.
+  // This field supersedes the ``party`` field. The effective set of parties on whose behalf the command
+  // should be executed is the union of all parties listed in ``party`` and ``act_as``, which must be non-empty.
+  // Each element must be a valid PartyIdString (as described in ``value.proto``).
+  // Optional
+  repeated string act_as = ;
+
+  // Set of parties on whose behalf (in addition to all parties listed in ``act_as``) contracts can be retrieved.
+  // This affects Daml operations such as ``fetch``, ``fetchByKey``, ``lookupByKey``, ``exercise``, and ``exerciseByKey``.
+  // Note: A participant node of a Daml network can host multiple parties. Each contract present on the participant
+  // node is only visible to a subset of these parties. A command can only use contracts that are visible to at least
+  // one of the parties in ``act_as`` or ``read_as``. This visibility check is independent from the Daml authorization
+  // rules for fetch operations.
+  // If ledger API authorization is enabled, then the authorization metadata must authorize the sender of the request
+  // to read contract data on behalf of each of the given parties.
+  // Optional
+  //  repeated string read_as = 13;
```

Comment:
just delete this?

Reply:
The goal was to be able to quickly spot the differences between transfer and standard command submission

Reply:
Yes, let's keep this for a while, but we will clean it up.